File: springboot/src/main/java/br/com/zup/springboot/PersonRunner.java
package br.com.zup.springboot;

import org.springframework.boot.CommandLineRunner;
import org.springframework.stereotype.Component;

import java.util.logging.Logger;

@Component
public class PersonRunner implements CommandLineRunner {
    private static final Logger LOGGER = Logger.getAnonymousLogger();

    private final PersonService service;

    public PersonRunner(PersonService service) {
        this.service = service;
    }

    @Override
    public void run(String[] args) {
        LOGGER.info(() -> service.readPerson(12345678900L).toString());
        LOGGER.info(() -> String.valueOf(service.createPerson(new Person(12345678900L, "Alguém", 10L))));
        LOGGER.info(() -> service.readPerson(12345678900L).toString());
        LOGGER.info(() -> String.valueOf(service.updatePerson(new Person(12345678900L, "Alguém2", 11L))));
        LOGGER.info(() -> service.readPerson(12345678900L).toString());
    }
}

File: springboot-kotlin/src/main/kotlin/br/com/zup/springboot/kotlin/PersonRunner.kt
package br.com.zup.springboot.kotlin

import org.springframework.boot.CommandLineRunner
import java.util.logging.Logger

class PersonRunner(val service: PersonService) : CommandLineRunner {
    val LOGGER = Logger.getAnonymousLogger()

    override fun run(vararg args: String?) {
        LOGGER.info { service.readPerson(12345678900)?.toString() ?: "NOT FOUND!!!" }
        service.createPerson(Person(12345678900, "Alguém", 10))
        LOGGER.info { service.readPerson(12345678900)?.toString() ?: "NOT FOUND!!!" }
        service.updatePerson(Person(12345678900, "Alguém 2", 12))
        LOGGER.info { service.readPerson(12345678900)?.toString() ?: "NOT FOUND!!!" }
        LOGGER.info { service.deletePerson(12345678900).toString() }
        LOGGER.info { service.readPerson(12345678900)?.toString() ?: "NOT FOUND!!!" }
    }
}
